packages/entities/entities-upstreams-targets/src/components/UpstreamsFormActiveHealthCheck.cy.ts

```diff
@@ -49,7 +49,7 @@ describe('<UpstreamsFormActiveHealthCheck/>', () => {
       },
     })
 
-    cy.getTestId('active-healthcheck-http-path').type('http-path')
+    cy.getTestId('active-healthcheck-http-path').type('http-path', { waitForAnimations: false })
```

Comment:
For PR review: not sure why but these assertions started to fail with reason "element is still animating". This helped, not sure if it's ideal solution though.